ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -1893,6 +1893,71 @@ class SignalProducerSpec: QuickSpec {
 				expect(started) == false
 			}
 		}
+		
+		describe("SignalProducer.merge") {
```

Comment:
I think this (and `describe("Signal.merge")` in `SignalSpec.swift`) should be defined in `FlattenSpec.swift`.